pkg/version/version.go

```diff
@@ -0,0 +1,7 @@
+package version
+
+var (
+	// Version provides the operators version
```

Comment:
just curious, how do we define the versions here?

Reply:
We don't update the versions actually. Just referred to one of the conventions that followed for other operators. Something which was just continued from older version of operators. Eg. https://github.com/openshift/managed-upgrade-operator/blob/master/version/version.go#L5 

Though, agreed that it doesn't add any value to keep it 1.0.0 always. Might be better to reflect the git commit perhaps to know which version is running but not something we have across the operators for now.